extensions/wikia/AdEngine/js/provider/btfBlocker.js

```diff
@@ -0,0 +1,90 @@
+/*global define*/
+define('ext.wikia.adEngine.provider.btfBlocker', [
```

Comment:
Have you checked mercury on consecutive pages? Probably we should reset variables on each pageview (?)